src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {!age || <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">{isMarried ? `${partnerName} is my ${partner}` : 'I am not married'}</p>
```

Comment:
Readabily is everything. Format ternary operator operands correctly - move each operand to the separate line:
```
 <p className="Person__partner">
      {isMarried
        ? `${partnerName} is my ${wifeOrHusband}`
        : 'I am not married'
      }
</p>
```